releases/release-1.35/release-notes/maps/pr-134948-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 134948
+releasenote:
+  text: Promoted `PodObservedGenerationTracking` to general availability.
```

```suggestion
  text: Promoted `PodObservedGenerationTracking` to GA.
```

Since we are using `GA` everywhere else